### ISSUE TITLE
NPM gulp-util deprecation warning

### ISSUE THREAD

**Issue by jimblue** [mjmlio/gulp-mjml#44]
Hi,

When running NPM install I get the following warning:

```bash
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
```

So I've run `npm ls gulp-util`  to get the list of plugins dependent on it, here is the result:

![Screenshot 2019-06-22 at 16 17 36](https://user-images.githubusercontent.com/10623367/59965030-449e9a80-9509-11e9-931e-2c56d058eda5.png)

But on my package.json I have only the following:

```json
"devDependencies": {
    "gulp": "4.0.2",
    "gulp-mjml": "3.0.0",
    "mjml": "4.4.0-beta"
}
```

I guess it's because actual **gulp-mjml 3** NPM release still use **gulp3**:

https://github.com/mjmlio/gulp-mjml/blob/9047d8d05f2df18a4f3e53fc6959bdf0ae1b0f15/package.json#L12

Whereas the actual master branch of **gulp-mjml** use **gulp4**:

https://github.com/mjmlio/gulp-mjml/blob/6093ee7f005c8c53286399d313ea2aac312f2a2b/package.json#L15

Can you release a new NPM version to fix that?

Thank you

**Comment by jimblue** [comment]
I've found others deprecations warnings also coming from **gulp-mjml** because of **gulp-3** dependency:

```bash
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
```

![Screenshot 2019-06-22 at 16 33 23](https://user-images.githubusercontent.com/10623367/59965290-4c137300-950c-11e9-967d-6bbb08f2c3f7.png)

```bash
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
```

![Screenshot 2019-06-22 at 16 36 13](https://user-images.githubusercontent.com/10623367/59965291-4c137300-950c-11e9-9df1-97b38620aa8e.png)

```bash
npm WARN deprecated natives@1.1.6: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.
```

![Screenshot 2019-06-22 at 16 36 53](https://user-images.githubusercontent.com/10623367/59965292-4c137300-950c-11e9-9362-51ae13568b32.png)

**Comment by jimblue** [comment]
Just a friendly ping 😄 !
Could you please update gulp-mjml dependencies to solve this?
Thank you

**Comment by iRyusa** [comment]
I've just published it today, and did upgrade `through2` to the latest version too.

Let me know if there's anything broken, it seems to work fine with the example file.

**Comment by jimblue** [comment]
awesome! the new release fix it! thank you @iRyusa 🎉

**Comment by iRyusa** [comment]
It looks like there’s an issue and we have to release a 4.0.1 asap cf #46 :(

My bad.

**Comment by kmcb777** [comment]
@jimblue The fix is released, it's actually 4.0.2 (my bad)

**Comment by jimblue** [comment]
I made the upgrade directly to latest before trying! So I didn't found any bug while testing it... But anyway thanks to ping me @kmcb777 !
